<launch>
    <node pkg="rosbag" type="record" name="recorder" args="--all -O $(find reflex)/bagfiles/latest_takktile_run"/>

	<param name="default_contact_threshold" value="20" />
	<param name="yaml_dir" value="$(find reflex_driver)/yaml" />
	<rosparam file="$(find reflex_driver)/yaml/motor_constants.yaml" command="load"/>
	<rosparam file="$(find reflex_driver)/yaml/finger_calibrate.yaml" command="load" ns = "hand1"/>
	<rosparam file="$(find reflex_driver)/yaml/tactile_calibrate.yaml" command="load" ns = "hand1"/>

	<node name="hand1_reflex_driver_node" pkg="reflex_driver" type="hand1_reflex_driver_node" output="screen" required="true"/>
</launch>
